src/backend/dx11/src/device.rs

```diff
@@ -776,11 +791,7 @@ impl Device {
             Windowed: TRUE,
             // TODO:
             SwapEffect: dxgi::DXGI_SWAP_EFFECT_DISCARD,
-            Flags: if config.present_mode == window::PresentMode::IMMEDIATE {
-                dxgi::DXGI_SWAP_CHAIN_FLAG_ALLOW_TEARING
```

Comment:
why are we no longer allowing tearing?

Reply:
That flag doesn't apply to `DXGI_SWAP_EFFECT_*` swap effects, only `DXGI_SWAP_EFFECT_FLIP*`. If used with non-flip effects, raises INVALID_ARG. This was the cause of gfx-rs/wgpu#1005